### DISCUSSION TITLE
Release of PDFsharp 6.2?

### DISCUSSION THREAD

**Discussion by PankajMishra88** [empira/PDFsharp#209] [Q&A]
Hello, I hope you’re doing well. May I ask when you’re expecting to release PDFsharp 6.2? Thank you!

**Comment by PankajMishra88** [comment]
@ThomasHoevel Can you help please?

**Reply by ThomasHoevel** [reply]
Sorry, but I cannot help.

PDFsharp 6.2 Preview 2 was released on Tuesday, December 10.
Now we're waiting for feedback.

**Comment by PankajMishra88** [comment]
> Hello, I hope you’re doing well. May I ask when you’re expecting to release PDFsharp 6.2? Thank you!

Hello, I hope you're doing well.

Could you please let me know the expected release date for PDFsharp 6.2? I noticed that Preview Version 3 is currently available; do we have an idea of how many additional preview versions are planned before the final 6.2 release?

Thank you!

**Reply by nkosi23** [reply]
@ThomasHoevel Any insights?

**Comment by PDFsharp-Team** [comment]
PDFsharp 6.2 final was released.
